cmd/crc/cmd/config/config.go

```diff
@@ -61,10 +42,44 @@ to the options that you set when you run the 'crc start' command.
 Configurable properties (enter as SUBCOMMAND): ` + "\n\n" + configurableFields()
 }
 
+func isPreflightKey(key string) bool {
+	return strings.HasPrefix(key, "skip-") || strings.HasPrefix(key, "warn-")
+}
+
+func less(lhsKey, rhsKey string) bool {
```

Comment:
It really took me sometime to understand what this function do, better to put a comment on the function to make sure next time we don't spend much time to figure out what it actually do :)